tests/controllers/management.test.ts

```diff
@@ -269,22 +253,77 @@ describe("DataController", () => {
           },
         },
       };
-      await edcClient.management.createAsset(
+      await edcClient.management.createAsset(context, assetInput);
+
+      // when
+      const assetDataAddress = await edcClient.management.getAsseDataAddress(
         context,
-        assetInput,
+        assetInput.asset.properties["asset:prop:id"],
       );
 
+      // then
+      expect(assetDataAddress).toHaveProperty("properties");
```

Comment:
Can we also check if "properties" is the same as the input above, please?

Is the outcome only `{ "properties": { ... } }` or there are other fields?

Reply:
The only outcome should be properties and you just made me realize that there is a mistake in the response definition because `properties` should be required (unlike `DataAddress` in `Asset`). I will fix it and update the tests accordingly.
Screenshot and [link](https://app.swaggerhub.com/apis/eclipse-edc-bot/management-api/0.0.1-SNAPSHOT#/Asset/getAssetDataAddress) for reference:
<img width="437" alt="Screenshot 2023-04-11 at 9 54 46 AM" src="https://user-images.githubusercontent.com/61057075/231108588-e6ab087d-5460-49c2-8aa1-518f3c9be195.png">